static/js/upload.js

```diff
@@ -19,7 +19,7 @@ export function feature_check($upload_button) {
 }
 
 export function get_translated_status(file) {
-    const status = $t({defaultMessage: "Uploading {filename}…"}, {filename: file.name});
+    const status = $t({defaultMessage: "Uploading {filename}"}, {filename: file.name});
```

Comment:
@alya I just wanted to check that removing the `...` is something we want to do -- that appears in the Figma design but I'm not sure it was explicitly discussed.